conformance/runcheck.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Why do we need this? Isn't it just a copy?

Reply:
fixed. Only difference was how we treat panics. (In normal tests we validate both if a panic happened and if the panic message is correct. But in conformance we check only if panic happened because our panic messages are not the same as jBallerina (Same is true for errors as well)